### DISCUSSION TITLE
[TOOL]AzerothCore Module Clone Script

### DISCUSSION THREAD

**Discussion by stoudtlr** [azerothcore/azerothcore-wotlk#3910] [Tools]
I created a script for simplifying cloning/updating modules.  It searches the AzerothCore repo and shows all available modules in a popup with checkboxes.  Modules that already exist in your modules folder are selected by default.  Select the ones you want and they are all cloned and/or updated.
![Imgur](https://i.imgur.com/kXeNNhK.png)
Instructions and code available here:

https://github.com/stoudtlr/AzerothCore-Scripts

**Comment by BarbzYHOOL** [comment]
i think it should be proposed here http://www.azerothcore.org/catalogue/ looks rad!

**Comment by FrancescoBorzi** [comment]
nice idea, is it cross-platform ?

**Comment by stoudtlr** [comment]
cross-platform....maybe? Microsoft released powershell core for linux but I haven't used it. I'm not sure if powershell core includes everything that's needed for the form to display correctly.  Maybe someone that uses linux could try it out?

**Comment by Yehonal** [comment]
I'm going to develop a similar tool in NodeJS that can be easily integrated with and electron GUI

**Comment by Yehonal** [comment]
Nice tool however! looks great

**Comment by Yehonal** [comment]
The visual impact of this simple tool is something people are looking for
